svc/pkg/monolith/standalone/worker/src/main.rs

```diff
@@ -7,18 +7,6 @@ fn main() -> GlobalResult<()> {
 async fn start() -> GlobalResult<()> {
 	let pools = rivet_pools::from_env("monolith-worker").await?;
 
-	tokio::task::Builder::new()
-		.name("monolith_worker::health_checks")
-		.spawn(rivet_health_checks::run_standalone(
-			rivet_health_checks::Config {
-				pools: Some(pools.clone()),
-			},
-		))?;
-
-	tokio::task::Builder::new()
-		.name("monolith_worker::metrics")
-		.spawn(rivet_metrics::run_standalone())?;
-
```

Comment:
> I opted to remove the code from lib and keep the one in main, since that seems to be what most other services do.

I think this is the opposite

Reply:
Oops, I swapped them around but forgot to commit 👍 [2baafcd](https://github.com/rivet-gg/rivet/pull/212/commits/2baafcd63aa27ce9a24a90c7cda9582e13aea258)